### DISCUSSION TITLE
should  a new release be created? the latest one is released two years ago

### DISCUSSION THREAD

**Discussion by jeffrey4l** [davatorium/rofi#2029] [General]
pls release a new one

**Comment by DaveDavenport** [comment]
~we?~ title been edited.

I don't have a lot of time atm. Help, "we", would be appreciated.

**Reply by jeffrey4l** [reply]
yes. we = any onf of the project owner or maintainer

pls help to release a new one, thanks a lot.

**Reply by DaveDavenport** [reply]
Why not you? anybody can help.

**Reply by jeffrey4l** [reply]
doesn't only owner has the permission to create new tag ? @DaveDavenport

**Reply by DaveDavenport** [reply]
A release is a lot more then 'create new tag'.  If it was that simple, we would have a new release by now.

**Reply by QiangF** [reply]
@DaveDavenport maybe just put a pre-compiled binary in the release page?

**Reply by DaveDavenport** [reply]
Pre-compilled for what?

**Reply by QiangF** [reply]
Sorry, I may be wrong. I thought rofi is portable.

**Reply by DaveDavenport** [reply]
Not sure what you mean with 'portable'.  Can you explain?

**Reply by QiangF** [reply]
Like appimage, compile once and work everywhere :-)

**Reply by DaveDavenport** [reply]
That is not my experience with appimage, I am also not sure it would work well with rofi's drun/run mode. 
Beside that;  I don't have time to figure out how to make those, cross-compile multiple versions for different architectures, etc.
If I find it hard to find time to make a release, how would making a release + appimages be less work?

(This should not be stopping anybody else from making some.)

**Reply by sledgehammer999** [reply]
Can you share what's needed to make a release so others can help as much as they can?
For example even PR #1958 is still pending at the moment for many months.

**Reply by DaveDavenport** [reply]
Yes that PR has been pending and you can see I've been pushing updates to it. It will be merged when we do a release.

* Go over all the documentation and see if they are all uptodate and the new features are documented.
* These issues needs to be resolved or moved: https://github.com/davatorium/rofi/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.7.6
* Final update of the releasenotes.
